runtime/src/bank/fee_distribution.rs

```diff
@@ -125,9 +143,10 @@ impl Bank {
                     ("num_lamports", deposit, i64),
                     ("error", err.to_string(), String),
                 );
-                *burn = burn.saturating_add(deposit);
+                deposit
             }
-        }
+        };
+        burn_undepositable_fund
```

Comment:
I think it is better to remove this variable `burn_undepositable_fund` and just leave match returning. This name of the variable seems to document what does this function actually returns, thus I would add a comment to this function.

Reply:
Sounds reasonable. I'll merge this, do comments in separate PR.